.qmake.conf

```diff
@@ -0,0 +1,2 @@
+top_srcdir=$$PWD
```

Comment:
Any reason to have these in lowercase? I think I'd prefer them to be uppercase. We tend to use UPPERCASE FOR QMAKE VARIABLES.

Reply:
No, I just copied them from Qt's website.

**TOP_SRCDIR** and **TOP_BUILDDIR**?